<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LittleSunClass</class>
 <widget class="QWidget" name="LittleSunClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LittleSun</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>338</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
border-image: url(:/login/image/login/background_a.png);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="passWordLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>176</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/login/image/login/password.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="passWordEdit">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>169</y>
      <width>164</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/login/image/login/input_box_a.png);
font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="userNameEdit">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>123</y>
      <width>164</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/login/image/login/input_box_a.png);
font: 8pt &quot;微软雅黑&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="userNameLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/login/image/login/username.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>220</y>
      <width>133</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/login/image/login/land.png);
}

QPushButton:hover{
border-image: url(:/login/image/login/landing.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="narrowBtn">
    <property name="geometry">
     <rect>
      <x>396</x>
      <y>0</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/login/image/login/j.png);
}

QPushButton:hover{
border-image: url(:/login/image/login/j_a.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-image: url(:/login/image/login/x.png);
}

QPushButton:hover{
border-image: url(:/login/image/login/x_a.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="test">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>userNameEdit</tabstop>
  <tabstop>passWordEdit</tabstop>
  <tabstop>loginBtn</tabstop>
 </tabstops>
 <resources>
  <include location="littlesun.qrc"/>
 </resources>
 <connections/>
</ui>
